tests/authTests/DeleteSessionTests.py

```diff
@@ -0,0 +1,61 @@
+import json
+from unittest import main
+import requests
+
+from TestConfiguration import TestConfiguration
+
+class DeleteSessionTests(TestConfiguration):
+
+   def setUp(self):      
+      registerUrl = self.baseUrl + '/users'
+      registerBody = {
+         'firstName' : 'Dave',
+         'lastName' : 'Janzen',
+         'email' : 'dave.janzen18@gmail.com',
+         'password' : 'Stockd2g'
+      }
+      registerResponse = requests.post(url=registerUrl, data=json.dumps(registerBody), headers=self.headers)
+
+      loginUrl = self.baseUrl + '/users/session'
+      loginBody = {
+         'email' : 'dave.janzen18@gmail.com',
+         'password' : 'Stockd2g'
+      }
+      loginResponse = requests.post(url=loginUrl, data=json.dumps(loginBody), headers=self.headers)
```

Comment:
The idea is that, login tests have been completed at this point so we don't have to check for that. But, let's talk about this for a hot second because I think you make a valid point.